src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -5,15 +5,19 @@ This section describes how to use the Prometheus monitoring system to collect, s
 
 .Prerequisites
 
-* {prod-short} is exposing metrics on port `8087`. See xref:enabling-and-exposing-{prod-id-short}-metrics_{context}[].
+* {prod-short} is exposing metrics on port `8087`.
+ifeval::["{project-context}" == "che"]
+See link:https://www.eclipse.org/che/docs/pages/che-7/administration-guide/proc_enabling-and-exposing-che-metrics.html[Enabling and exposing {prod-id-short} metrics].
+endif::[]
+ifeval::["{project-context}" == "crw"]
+See link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.0/html/administration_guide/monitoring-codeready-workspaces#enabling-codeready-workspaces-metrics-collections[Enabling {prod-short} metrics collections].
```

Comment:
This is upstream -- so, no downstream links in the source code. Please, use attributes for the URLs.

Reply:
It is in `ifeval` so it is not rendered for the upstream docs